internal/syscall.go

```diff
@@ -68,6 +69,48 @@ func BPF(cmd BPFCmd, attr unsafe.Pointer, size uintptr) (uintptr, error) {
 	return r1, err
 }
 
+type BPFProgLoadAttr struct {
+	ProgType           uint32
+	InsCount           uint32
+	Instructions       Pointer
+	License            Pointer
+	LogLevel           uint32
+	LogSize            uint32
+	LogBuf             Pointer
+	KernelVersion      uint32     // since 4.1  2541517c32be
```

Comment:
Not sure if worth keeping the comments, since the latest fields don't have it and I doubt there's much motivation to keep the list up-to-date (but that's not my call to make).

Reply:
If we choose to remove the comments I'd say we can do it in another PR, likewise if we'd chose to add some comments to keep the fields up-to-date.

Reply:
No objection